### ISSUE TITLE
Broken reloading after first reload

### ISSUE THREAD

**Issue by dsebastien** [BrowserSync/browser-sync-client#28]
When I start BrowserSync, the client gets loaded correctly in Google Chrome (v46), but after a first browserSync.reload (initiated from a gulp.watch, I get the following error, after which the reloading doesn't work anymore:
```
browser-sync-client.2.9.11.js:129 Uncaught TypeError: o.getWindow(...).scrollTo is not a function
```

Any idea?

It seems to work normally under other browsers. I use version 2.9.x

**Comment by shakyShane** [comment]
Please recreate this issue with a small sample project. I cannot reproduce the bug you describe :)

**Comment by dsebastien** [comment]
Hi @shakyShane. I can't reproduce the issue anymore. It went away after I removed the following BrowserSync server init options: reloadDebounce, reloadDelay. It became weirder when adding those back didn't make the issue reappear. Let's just assume that whatever happened was due to my configuration alone.

FYI here's my config: https://github.com/dsebastien/modernWebDevBuild/blob/master/src/gulp/tasks/serve.js

Thanks!
